Change project fields to match Scylla (#46)
scylladb/csharp-driver: README.md

> # ScyllaDB C# Driver for Apache Cassandra

Reviewer: ```suggestion
# ScyllaDB C# Driver for Scylla
```

> ScyllaDB's fork of a modern, [feature-rich][features] and highly tunable C# client library for Apache Cassandra (2.0+) using Cassandra's binary protocol and Cassandra Query Language v3.

Reviewer: ```suggestion
ScyllaDB's fork of a modern, [feature-rich][features] and highly tunable C# client library for Scylla using Cassandra's binary protocol and Cassandra Query Language v3.
```

- Works with any cluster size
- [Linq2Cql][linq] and Ado.Net support

## Scylla features

> - shard awarness
> - tablet awareness
> - LWT support

Reviewer: ```suggestion
- Shard awarness
- Tablet awareness
- LWT prepared statements metadata mark
```
